### ISSUE TITLE
Failure on sending filenames with spaces

### ISSUE THREAD

**Issue by jdenoc** [jdenoc/travis-ci-discord-webhook#1]
Attempted to send an image with spaces in its filename.

Sample output from a failure:
```
$ ./image-send.sh $DUSK_SCREENSHOT_DIRECTORY $WEBHOOK_URL
tests/Browser/screenshots/failure-testClickingOnEntryTableEditButtonOfUnconfirmedEntry with data set "Expense"-0.png
basename: extra operand ‘data’
Try 'basename --help' for more information.
[Webhook]: Sending image via webhook.
curl: (6) Could not resolve host: with
[Webhook]: Unable to send image via webhook.
```

Due to the spaces in filename, and in this case, the word `with` is treated as the `$WEBHOOK_URL` value instead of the `$WEBHOOK_URL` value.

**Comment by jdenoc** [comment]
Offending line of code:
https://github.com/jdenoc/travis-ci-discord-webhook/blob/d837e187a32032b3e9b8fd2cae1ab8fa459ce4c1/image-send.sh#L75

`$filename` should be enclosed in `"` or something...

**Comment by jdenoc** [comment]
Resolved by #2
